test/fixtures/node_modules/pkgexports/package.json

```diff
@@ -2,7 +2,6 @@
   "exports": {
     ".": "./asdf.js",
     "./space": "./sp%20ce.js",
-    "./asdf": "./asdf.js",
```

Comment:
Any particular reason for removal or is this just redundancy?

Reply:
Just redundancy. I was trying to ensure full coverage when combining the test suites and got confused by the two entries. So I deleted the one I deemed redundant.